aspnetcore/fundamentals/app-state.md

```diff
@@ -1,236 +1,191 @@
 ---
-title: Managing Application State | Microsoft Docs
+title: Session and application state | Microsoft Docs
 author: rick-anderson
-description: 
-keywords: ASP.NET Core,
+description: Approaches to preserving application and user (session) state between requests.
+keywords: ASP.NET Core, Application state, session state, querystring, post
 ms.author: riande
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 1/14/2017
 ms.topic: article
 ms.assetid: 18cda488-0769-4cb9-82f6-4c6685f2045d
 ms.technology: aspnet
 ms.prod: aspnet-core
 uid: fundamentals/app-state
 ---
-# Managing Application State
+<!-- Review questions
 
->[!WARNING]
-> This page documents version 1.0.0-rc1 and has not yet been updated for version 1.0.0
+When does session clear the backing store? When the session expires? After you close the browser and the session cookie is deleted - what happens to your session data?
```

Comment:
Data is deleted from the backing store either when you call Session.Clear() or when the session expires in the data store. The server does not know when the browser is closed or the cookie is deleted.